assets/shared-components/ctaLink.jsx

```diff
@@ -0,0 +1,24 @@
+// ----- Imports ----- //
```

Comment:
Is `shared-components` standard React directory structure?  To me the `shared-` prefix is just noise since the directories position in the hierarchy makes that clear